src/css.js

```diff
@@ -133,21 +130,33 @@ function getWidthOrHeight( elem, dimension, extra ) {
 		val = "auto";
 	}
 
-	// Check for style in case a browser which returns unreliable values
-	// for getComputedStyle silently falls back to the reliable elem.style
-	valueIsBorderBox = valueIsBorderBox &&
-		( support.boxSizingReliable() || val === elem.style[ dimension ] );
+	isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
+	valueIsBorderBox = isBorderBox && support.boxSizingReliable();
+	offsetProp = "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 );
 
 	// Fall back to offsetWidth/offsetHeight when value is "auto"
 	// This happens for inline elements with no explicit setting (gh-3571)
 	// Support: Android <=4.1 - 4.3 only
 	// Also use offsetWidth/offsetHeight for misreported inline dimensions (gh-3602)
-	if ( val === "auto" ||
-		!parseFloat( val ) && jQuery.css( elem, "display", false, styles ) === "inline" ) {
-
-		val = elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ];
+	// Support: IE 9-11 only
+	// Also use offsetWidth/offsetHeight for when box sizing is unreliable
+	// Do not use offset for elements that don't have it (e.g. SVG)
+	if ( typeof elem[ offsetProp ] !== "undefined" &&
```

Comment:
Would moving the `typeof` check to the end of the `if` make sense to avoid computing it if not needed?

Reply:
It's my understanding that `typeof` doesn't cause a compute, but I'll double check that. Could also look at `hasOwnProperty`.

Reply:
You're right, it does cause a compute. I switched to `hasOwnProperty`, but I think I'll keep it at the front because I'm concerned that `jQuery.css( elem, "display", false, styles ) === "inline"` might be a little more expensive.

Reply:
Sigh, `hasOwnProperty` returned `false` in Chrome despite having the property on inline elements. I just did what you suggested.

Reply:
@mgol brilliant

Reply:
I like it. It brings it down to +6 as well.